.travis.yml

```diff
@@ -8,9 +8,10 @@ env:
 matrix:
     fast_finish: true
     include:
+        - php: 5.6
         - php: 7.0
           env: CHECK_PHP_SYNTAX="yes"
```

Comment:
There is chances the php 7.1 build is slightly faster than 7.0. I would move the `php-cs-fixer` check to this build :) WDYT ?